### PR TITLE
Remove settings from EventHubOptions

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs [hunk 1]
```diff
@@ -25,27 +25,20 @@ namespace Microsoft.Azure.WebJobs.EventHubs
 {
     public partial class EventHubOptions : Microsoft.Azure.WebJobs.Hosting.IOptionsFormatter
     {
-        public const string LeaseContainerName = "azure-webjobs-eventhub";
         public EventHubOptions() { }
         public int BatchCheckpointFrequency { get { throw null; } set { } }
         public Azure.Messaging.EventHubs.EventHubConnectionOptions ConnectionOptions { get { throw null; } set { } }
         public Microsoft.Azure.WebJobs.EventHubs.InitialOffsetOptions InitialOffsetOptions { get { throw null; } }
         public bool InvokeFunctionAfterReceiveTimeout { get { throw null; } set { } }
```
Comment (JoshLove-msft) [inline sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs:32]: Weren't we removing this too?

File: sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs [hunk 2]
```diff
@@ -25,27 +25,20 @@ namespace Microsoft.Azure.WebJobs.EventHubs
...
-        public Azure.Messaging.EventHubs.Processor.LoadBalancingStrategy LoadBalancingStrategy { get { throw null; } set { } }
         public System.TimeSpan LoadBalancingUpdateInterval { get { throw null; } set { } }
         public int MaxBatchSize { get { throw null; } set { } }
         public System.TimeSpan MaximumWaitTime { get { throw null; } set { } }
```
Comment (JoshLove-msft) [inline sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs:35]: And this?

File: sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs (hunk 1 above)
Comment (pakrym) [reply sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs:32]: Yes, in the next PR.

File: sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs (hunk 2 above)
Comment (pakrym) [reply sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/api/Microsoft.Azure.WebJobs.Extensions.EventHubs.netstandard2.0.cs:35]: Yes, in the next PR.

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs [hunk 3]
```diff
@@ -348,7 +348,7 @@ public class EventHubTestMultipleDispatchJobs
 
             private const int EventCount = 5;
             private static readonly object MessageLock = new object();
-            public static void SendEvents_TestHub(string input, [EventHub(TestHubName)] out EventData[] events)
+            public static void SendEvents_TestHub(string input, [EventHub(TestHubName, Connection = TestHubName)] out EventData[] events)
```
Comment (JoshLove-msft) [inline sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs:351]: Do we have a test using ConsumerGroup?

Comment (pakrym) [reply sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs:351]: Don't think so

Comment (JoshLove-msft) [reply sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs:351]: Consider adding one - you can create an EH with a consumer group in the existing EventHubScope

Comment (pakrym) [reply sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs:351]: OK

Comment (pakrym) [reply sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubApplicationInsightsTests.cs:351]: I added a test that uses CG passed through the configuration.

**Pull request merged**